src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -301,6 +301,14 @@ message SignedAddressAllocation {
     /// (Required) The IP address and metadata that was allocated.
     AddressAllocation addressAllocation = 1;
 
-    // (Required) This is an x509v3 signed version of the address allocation.
-    bytes signedAddressAllocation = 2;
-};
\ No newline at end of file
+    // authoritative publicKey is the key that came from the database
+    bytes authoritativePublicKey = 3;
+    // the hostPublicKey is the ephemeral public key
+    bytes hostPublicKey = 4;
+    // hostPublicKeySignature is the signature of the authoratative key signing the hostPublicKey
+    bytes hostPublicKeySignature = 5;
+    // serialized version of the addressAllocation message
+    bytes message = 6;
```

Comment:
Is the protobuf serialization not consistent enough to be relied on? Are we going to be able to check that the serialized message here and the `addressAllocation` above are not altered?

Reply:
Protobuf serialization is not guaranteed to be stable. See: https://developers.google.com/protocol-buffers/docs/encoding

> Do not assume the byte output of a serialized message is stable. This is especially true for messages with transitive bytes fields representing other serialized protocol buffer messages.

Reply:
Makes sense. Is the agent or VPC service going to validate that the signed payload matches the `addressAllocation` field?

Reply:
The VPC service will validate the value.